.changeset/long-bags-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystonejs/file-adapters': minor
```

Comment:
```suggestion
'@keystonejs/file-adapters': major
```

Any one using this will need to change how they provided: `accessKeyIdm` `secretAccessKey` and `region`, making this a breaking change.

We'll also need to inform people of this in the changeset with specific details on how to update if using those options.